src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2333,7 +2356,9 @@ def _update_addons(cmd,  # pylint: disable=too-many-branches,too-many-statements
                    dns_zone_resource_ids=None,
                    no_wait=False,  # pylint: disable=unused-argument
                    enable_syslog=False,
-                   data_collection_settings=None):
+                   data_collection_settings=None,
+                   ampls_resource_id=None,
+                   enable_high_log_scale_mode=False):
```

Comment:
It seems to me that these parameters are not really used (passed to ensure_container_insights_for_monitoring and create some resources) but are just discarded here. And it looks like the same thing happens in azure-cli.

Note there are two commands that can be used to enable an addon in aks-preview, namely `az aks enable-addons` and `az aks addon enable`. They have completely different code paths. In azure-cli, only `az aks enable-addons` is supported.

Reply:
> It seems to me that these parameters are not really used (passed to ensure_container_insights_for_monitoring and create some resources) but are just discarded here. And it looks like the same thing happens in azure-cli.
> 
> Note there are two commands that can be used to enable an addon in aks-preview, namely `az aks enable-addons` and `az aks addon enable`. They have completely different code paths. In azure-cli, only `az aks enable-addons` is supported.

removed unused parameters here and as well azure-cli.